.dockerignore

```diff
@@ -1,9 +1,9 @@
-**/.git
-.github/
-.vscode/
-.deno/
-test/
-docs/
-contrib/
-**/node_modules
-private.key
+**/*
```

Comment:
Wondering if anyone uses local [plugins](https://github.com/CycloneDX/cdxgen/tree/master/plugins). The idea was binary plugins built locally and placed in that folder would have precedence over the `node_modules` one.

Reply:
Ah, that's what the 'plugins'-folder is for? I can add that back in, if you want.